lib/simplifyExpression/basicsSearch/reduceMultiplicationByZero.js

```diff
@@ -1,25 +1,21 @@
+const {query: q} = require('math-nodes');
```

Comment:
I used `q` so that I didn't have to break as many lines.  I can change it to `query` if you'd like.